mesonbuild/interpreter/interpreter.py

```diff
@@ -2851,7 +2851,7 @@ def validate_installable_file(fpath: Path) -> bool:
             # subproject files, as long as they are scheduled to be installed.
             if validate_installable_file(norm):
                 return
-        norm = Path(srcdir, subdir, fname).resolve()
+        norm = Path(os.path.abspath(Path(srcdir, subdir, fname)))
```

Comment:
Yet again, pathlib proves that it sucks horribly at anything other than PurePath.